### PR TITLE
Upgrade Bokeh to Verison 3.3.0

### PR DESCRIPTION
**Pull request by mfixstsci**
Remove bokeh template files/imports and bump bokeh version from `2.4.3 --> 3.3.0`

### CODE REVIEW

File: jwql/website/apps/jwql/bokeh_containers.py [hunk 1]
```diff
@@ -444,7 +445,8 @@ def standard_monitor_plot_layout(instrument, plots):
 
     # Combine full frame and subarray aperture lists
     full_list = full_frame_lists + subarr_lists
-
+    print(full_frame_lists)
```
Comment (mfixstsci) [inline jwql/website/apps/jwql/bokeh_containers.py]: Need to remove these lines.

**Comment by mfixstsci** [comment]
@BradleySappington @bhilbert4 All of the tests are passing, I caught all of the little imports and updates to templating etc. Ready for review.

File: jwql/website/apps/jwql/bokeh_containers.py [hunk 2]
```diff
@@ -24,7 +24,8 @@
 
 from bokeh.embed import components
 from bokeh.layouts import layout
-from bokeh.models.widgets import Tabs, Panel
+# from bokeh.models.widgets import Tabs, Panel
```
Comment (bhilbert4) [inline jwql/website/apps/jwql/bokeh_containers.py]: This can be deleted, right?

File: jwql/bokeh_templating/factory.py [hunk 3]
```diff
@@ -36,8 +36,8 @@
 
 # Figures get their own constructor so we remove references to Figures from
 # the keyword maps.
-Figure = mappings.pop("Figure")
-del sequences["figure"]
+Figure = mappings.pop("figure")
+# del sequences["figure"]
```
Comment (BradleySappington) [inline jwql/bokeh_templating/factory.py]: Is there a reason you are commenting and not deleting?

**Review by BradleySappington — COMMENTED** [review]
Looks good.  Does this no longer support versions before 3?

File: jwql/bokeh_templating/factory.py [hunk 4]
```diff
@@ -36,8 +36,8 @@
 
 # Figures get their own constructor so we remove references to Figures from
 # the keyword maps.
-Figure = mappings.pop("Figure")
-del sequences["figure"]
+Figure = mappings.pop("figure")
```
Comment (bhilbert4) [inline jwql/bokeh_templating/factory.py:39]: This shouldn't be used anywhere anymore, right? Sometime in the near future, we can have a PR that deletes this file?

File: jwql/bokeh_templating/factory.py (hunk 3 above)
Comment (mfixstsci) [reply jwql/bokeh_templating/factory.py]: Whoops, was doing that while testing. I can remove it.

**Comment by mfixstsci** [comment]
> Looks good. Does this no longer support versions before 3?

Exactly, we are ending support for bokeh versions less than 3.0

**Review by bhilbert4 — APPROVED** [review]
LGTM. Just a couple of cosmetic questions.

File: pyproject.toml [hunk 5]
```diff
@@ -17,7 +17,7 @@ dependencies = [
   "asdf",
   "astropy",
   "astroquery",
-  "bokeh<3",
+  "bokeh",
```
Comment (BradleySappington) [inline pyproject.toml]: this must be >= 3 if we no longer support 2

File: jwql/bokeh_templating/factory.py (hunk 4 above)
Comment (mfixstsci) [reply jwql/bokeh_templating/factory.py:39]: I agree -- this is still pretty fragile and was getting scooped up in our tests. I think we phase it our in one of our releases in the future.

**Pull request merged**